bin/jsii-release-maven

```diff
@@ -90,7 +91,7 @@ import_gpg_key() {
     export GPG_TTY=$(tty)
 
 
-    if [ -n "${MAVEN_GPG_PRIVATE_KEY}" ]; then
+    if [ -n "${MAVEN_GPG_PRIVATE_KEY-}" ]; then
```

Comment:
That's what I had intended to change it to. Not sure how my testing started working after that, so I'll look into it more.

Reply:
Going to switch to that since it's a more complete check. Just having `-` worked because it was still a type of expansion (See https://stackoverflow.com/a/32675207)